### PR TITLE
- Updated Example Code to support Flutter 3.3.0 & Android v2 Embedding 

### PR DESCRIPTION
**Pull request by milansurelia**
- Updated Example Code to support Android v2 Embedding
- Support for Flutter 3.3

#22

### CODE REVIEW

**Comment by milansurelia** [comment]
The PR will also resolve [this](https://github.com/cb-cloud/flutter_in_app_notification/issues/22) Issue

File: example/android/build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```
Comment (Kurogoma4D) [inline example/android/build.gradle]: Did it actually work? In my environment, version 4.1.0 is required in Flutter v3.3.

```groovy
classpath 'com.android.tools.build:gradle:4.1.0'
```

File: example/android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```
Comment (Kurogoma4D) [inline example/android/gradle/wrapper/gradle-wrapper.properties]: Same to above.

```groovy
distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

**Review by Kurogoma4D — COMMENTED** [review]
Thanks for contribute! almost LGTM, but I could not confirm working. Please check these comments.

And, to support Flutter 3.3, updating `compileSdkVersion` and `targetSdkVersion` is may required.

```groovy
android {
    compileSdkVersion 31
    sourceSets {
        main.java.srcDirs += 'src/main/kotlin'
    }
    lintOptions {
        disable 'InvalidPackage'
    }
    defaultConfig {
        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
        applicationId "com.cbcloud.in_app_notification_example"
        minSdkVersion 21
        targetSdkVersion 31
        versionCode flutterVersionCode.toInteger()
```

**Comment by milansurelia** [comment]
@Kurogoma4D  Viewed your change Request, I will soon update the changes.

File: example/android/build.gradle (hunk 1 above)
Comment (milansurelia) [reply example/android/build.gradle]: @Kurogoma4D  I have Updated Code
Kindly Review & merge the PR.

File: example/android/gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (milansurelia) [reply example/android/gradle/wrapper/gradle-wrapper.properties]: @Kurogoma4D I have Updated Code
Kindly Review & merge the PR.

**Comment by milansurelia** [comment]
@Kurogoma4D I have Updated Code
Kindly Review & merge the PR.

**Review by Kurogoma4D — APPROVED** [review]
Good work, thanks! 👍

**Pull request merged**